docs/develop/worker-performance.mdx

```diff
@@ -37,22 +37,7 @@ The Workflow Cache is created and shared between all the workers. It's designed
 
 These options limit the resource consumption of the in-memory Workflow cache. Workflow cache options are shared between all Workers, because the Workflow cache is something that has to do with the resource consumption of the whole host, like memory and the total amount of threads, and should be limited per JVM.
 
-## Monitor Task Queue backlog metrics {#task-queue-metrics}
-
-A [Task Queue](https://docs.temporal.io/workers#task-queue) is a lightweight, dynamically allocated queue.
-[Worker Entities](https://docs.temporal.io/workers#worker-entity) poll the queue for [Tasks](https://docs.temporal.io/workers#task).
-The Temporal Service dynamically creates different [Task Queue types](/workers#task-queue) including Activity Task Queues, Workflow Task Queues, and Nexus Task Queues.
-These Task Queue types route their Tasks to Workers for Task completion.
-
-With an accurate estimate of backlog Tasks, you can determine the optimal number of Workers to deploy.
-Balance your Worker count with the number of Tasks to achieve the best performance.
-This approach minimizes Task backlog saturation and reduces idle Workers.
-
-Task Queue metrics provide numerical insights into your Task Queue activity and backlog characteristics.
-Use these metrics to tune your production deployments.
-Evaluate your Worker loads and assess whether you need to scale up or reduce your Worker deployment.
-
-### Task Queue types {#task-queue-types}
+## Task Queue types {#task-queue-types}
```

Comment:
Looking at the side nav now, "Task Queue types" as a main level heading isn't making sense unfortunately. As a user, I care about the "Available Task Queue Information"... the fact that there are different "Task Queue types" helps my understanding but isn't useful in and of itself.

Reply:
Revised per suggestion.